msg-socket/src/rep/socket.rs

```diff
@@ -137,13 +145,14 @@ where
             auth_tasks: JoinSet::new(),
             compressor: self.compressor.take(),
             conn_tasks: FuturesUnordered::new(),
+            control_rx,
             span,
         };
 
-        tokio::spawn(backend);
-
+        self._driver_task = Some(tokio::spawn(backend));
```

Comment:
what's the use case for keeping the handle inside the driver?

Reply:
There is no particular use case, just keeping track of tasks spawned. Can be used however to check whether the backend started and run assumptions against it.